cmd/root.go

```diff
@@ -55,17 +55,17 @@ func init() {
 	rootCmd.PersistentFlags().String("signature", "", "Rekor signature")
 	viper.BindPFlag("signature", rootCmd.PersistentFlags().Lookup("signature"))
 
-	rootCmd.PersistentFlags().String("public_key", "", "Rekor publickey")
-	viper.BindPFlag("public_key", rootCmd.PersistentFlags().Lookup("public_key"))
+	rootCmd.PersistentFlags().String("public-key", "", "Rekor publickey")
+	viper.BindPFlag("public-key", rootCmd.PersistentFlags().Lookup("public-key"))
 
-	rootCmd.PersistentFlags().String("artifact_path", "", "Rekor artifact path")
-	viper.BindPFlag("artifact_path", rootCmd.PersistentFlags().Lookup("artifact_path"))
+	rootCmd.PersistentFlags().String("artifact-path", "", "Rekor artifact path")
+	viper.BindPFlag("artifact-path", rootCmd.PersistentFlags().Lookup("artifact-path"))
 
-	rootCmd.PersistentFlags().String("artifact_url", "", "Rekor artifact url")
-	viper.BindPFlag("artifact_url", rootCmd.PersistentFlags().Lookup("artifact_url"))
+	rootCmd.PersistentFlags().String("artifact-url", "", "Rekor artifact url")
+	viper.BindPFlag("artifact-url", rootCmd.PersistentFlags().Lookup("artifact-url"))
 
-	rootCmd.PersistentFlags().String("artifact_sha", "", "Rekor artifact sha")
-	viper.BindPFlag("artifact_sha", rootCmd.PersistentFlags().Lookup("artifact_sha"))
+	rootCmd.PersistentFlags().String("artifact-sha", "", "Rekor artifact sha")
+	viper.BindPFlag("artifact-sha", rootCmd.PersistentFlags().Lookup("artifact-sha"))
```

Comment:
Similar to my latest Gosec patch on the server side, perhaps consolidate into a single BindPFlag call at the end?

Reply:
let's do that in a separate PR , keep the commits a bit more siloed to the change.